src/jit/emitxarch.cpp

```diff
@@ -4812,6 +4922,26 @@ void emitter::emitIns_SIMD_R_R_R(instruction ins, regNumber reg, regNumber reg1,
         emitIns_R_R(ins, emitTypeSize(simdtype), reg, reg2);
     }
 }
+
+void emitter::emitIns_SIMD_R_R_R_I(
+    instruction ins, regNumber reg, regNumber reg1, regNumber reg2, ssize_t ival, var_types simdtype)
+{
+#pragma warning(disable : 4244)
```

Comment:
What is raising this warning? I added a similar function for my PR and didn't see this.

Reply:
The warning is raised due to conversion of `ssize_t` to `int`. `ssize_t` is defined as `long long` and cast to `int` results in potential loss of value what raises warning.

This is part of the code handling immediate operands, it requires new declaration of `GenTreeJitIntrinsics` which defines `gtImmValue` and several new functions and features supporting immediate value handling

Reply:
Looks like `ival`'s type needs to be `int`, not `ssize_t`. Anyway, if there's a good reason to use `ssize_t` here then such warnings are normally silenced by using explicit casts, not by disabling warnings with pragmas.